backend/src/test/java/com/woowacourse/levellog/acceptance/AcceptanceTest.java

```diff
@@ -125,4 +119,45 @@ protected RestAssuredResponse login(final String nickname) {
 
         return null;
     }
+
+    protected RestAssuredResponse requestCreateTeam(final String title, final String token,
+                                                    final int interviewerNumber, final List<Long> participantIds) {
```

Comment:
```suggestion
    protected RestAssuredResponse requestCreateTeam(final String title, final String token, final int interviewerNumber,
                                                    final List<Long> participantIds) {
```
밑에 오버로딩된 requestCreateTeam랑 맞춰주는 게 둘이 비교하기 좋을 거 같아용

Reply:
해당 메서드를 사용하는 쪽에서 계속 List.of()를 사용하고있는데요. 가변 인자로 변경하면 사용하는 쪽이 더 편하지 않을까요?
사용하는 클라이언트가 코드를 최대한 적게 쓰도록 해야 테스트 가독성을 높일 수 있고 픽스처로써 역할은 한다고 생각해요 😋

```suggestion
    protected RestAssuredResponse requestCreateTeam(final String title, final String token,
                                                    final int interviewerNumber, final Long... participantIds) {
```

Reply:
다른 테스트 픽스처와 동일하게 `requestCreateXXX` ➡️ `saveXXX` 은 어떨까요?

Reply:
List.of를 사용하는 이유는 가변 인자로 사용했을 때 int interviewerNumber와 participantIds가 혼동될 우려가 있다고 생각합니다.
따라서 List.of로 번거롭지만 이를 감싸줌으로써 participantIds 인것을 표현하기 위한 목적으로 작성했습니다~ 😊